### PR TITLE
WarningBox: hover cursor and disable selecting text

### LINKED ISSUE #2510: Settings page: mouse cursor in slider isn't a pointing hand
Expected mouse cursor for slider is a pointing hand (the same as shown in "Lock wallet on inactivity")

### LINKED ISSUE #2511: Text can be selected in warning boxes
image

### LINKED ISSUE #2512: Mouse cursor in warning links is not a pointing hand

### PR DESCRIPTION
**Pull request by selsta**
Fixes #2511
Fixes #2512

### CODE REVIEW

**Comment by rating89us** [comment]
You mean Fixes #2512

**Comment by selsta** [comment]
true, thanks

File: components/WarningBox.qml [hunk 1]
```diff
@@ -44,19 +44,22 @@ Rectangle {
...
-            textMargin: 0
             leftPadding: 4
             rightPadding: 18
             topPadding: 10
             bottomPadding: 10
-            readOnly: true
             onLinkActivated: root.linkActivated();
 
             selectionColor: MoneroComponents.Style.textSelectionColor
             selectedTextColor: MoneroComponents.Style.textSelectedColor
```
Comment (xiphon) [inline components/WarningBox.qml]: can drop these lines too

Comment (xiphon) [reply components/WarningBox.qml]: And maybe replace `TextArea` with `Text`

Comment (selsta) [reply components/WarningBox.qml]: updated

**Pull request merged**